ConsentViewController/Classes/SPSDK.swift

```diff
@@ -56,11 +57,24 @@ public extension SPSDK {
         loadMessage(forAuthId: authId, publisherData: pubData)
     }
 
-    func loadCCPAPrivacyManager(withId id: String, tab: SPPrivacyManagerTab = .Default) {
-        loadCCPAPrivacyManager(withId: id, tab: tab)
+    func loadCCPAPrivacyManager(withId id: String, tab: SPPrivacyManagerTab = .Default, useGroupPmIfAvailable: Bool = false) {
```

Comment:
@Nevazhnovu you first need to add:
```swift
loadCCPAPrivacyManager(withId: id, tab: SPPrivacyManagerTab, useGroupPmIfAvailable: Bool)
```
to `SPCCPA` protocol (the method is already there but it's missing the last parameter).

Then all you need to do in the extension is to call it with the default values (you don't need to add any particular concrete implementation). Eg.
```swift
func loadCCPAPrivacyManager(withId id: String, tab: SPPrivacyManagerTab = .Default, useGroupPmIfAvailable: Bool = false) {
    loadCCPAPrivacyManager(withId: id, tab: tab, useGroupPmIfAvailable: useGroupPmIfAvailable)
}
```
And do the same for GDPR.

PS.
We use the extension only because Swift Protocols don't allow for arguments with default argument. So we _extend_ it to add the method with default arguments.